### ISSUE TITLE
didn't find v1.20.16-lts.1 binaries for ubuntu

### ISSUE THREAD

**Issue by jinjiaw** [klts-io/kubernetes-lts#132]
### What happened?

I am following the guide https://klts.io/docs/install to have a try with [v1.20.16-lts.1](https://klts.io/docs/kubernetes/releases/v1.20/v1.20.16-lts.1/). But after adding the KLTS source package for ubuntu, i could only see version 1.10.13 to 1.18.20, and all the lts versioning is lts.0 which also confused me a little bit. 

```
# apt-cache show kubeadm | grep Version
Version: 1.18.20-lts.0
Version: 1.17.17-lts.0
Version: 1.16.15-lts.0
Version: 1.15.12-lts.0
Version: 1.14.10-lts.0
Version: 1.13.12-lts.0
Version: 1.12.10-lts.0
Version: 1.11.10-lts.0
Version: 1.10.13-lts.0
```

### What did you expect to happen?

should see KLTS versions after 1.18 in ubuntu source packages. 

### How can we reproduce it (as minimally and precisely as possible)?

```bash
cat << EOF > /etc/apt/sources.list.d/klts.list
deb [trusted=yes] https://dl.klts.io/deb stable main
EOF

apt-get update

# apt-cache show kubeadm | grep Version
```

**Comment by pacoxu** [comment]
```
cat << EOF > /etc/apt/sources.list.d/klts.list
deb [trusted=yes] https://raw.githubusercontent.com/klts-io/kubernetes-lts/deb-v1.20.16-lts.1 stable main
EOF

apt update

VERSION=1.20.16-lts.1
apt-get install kubeadm=${VERSION} kubelet=${VERSION} kubectl=${VERSION}
```
Current release pkgs are in `https://github.com/klts-io/kubernetes-lts/tree/deb-v1.XX.XX-lts.X/`

`dl.klts.io/deb` is not updated yet.

**Comment by wzshiming** [comment]
Yes, 1.20.16 this version has not been released officially, it has been rereleased 1.20.15-lts.2

/close
